alifd__next/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "alifd__next",
+  "filename": "next.min.js",
+  "version": "1.19.24",
+  "description": "A configurable component library for web built on React.",
+  "homepage": "https://github.com/alibaba-fusion/next",
+  "keywords": [
+    "fusion",
+    "fu`sion design",
+    "next",
+    "component",
+    "ui tookit",
+    "react",
+    "react-components",
+    "components",
+    "design",
+    "frontend"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/alibaba-fusion/next.git"
+  },
+  "license": "MIT",
+  "npmName": "@alifd/next",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "@alifd/next",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  }
```

```suggestion
  ]
```
Please remove the `autoupdate` attr, this is not the correct usage. Auto-updating with NPM is done via the `npmName` and `npmFileMap` attrs.